Species/Languages.md

```diff
@@ -1,49 +1,146 @@
+* **Giant:** The language created by the elder titans of old, and now spoken by the giants and their descendants. Often learned by scholars, or traders who would venture to giant settlements. Commonly learned by dwarves, especially those of the Sentinels. Real world mapping: Old Norse. 
+* **Halfing:** The secretive and rarely taught language of the halflings, made even more difficult for outsiders to learn given the expectation that speakers of this language will use telepathic cues for emphasis.  The halflings' language is preserved and learned via the racial consciousness; a single dialect covers all halflings across the world, even on diverse continents. The halflings rarely, if ever, teach their secretive language to outsiders. As far as outsiders know, there is no written form of halfling. Real world mapping: considering Igbo.
+* **Centaur:** The language of centaurs. Uses the Elvish alphabet. Rarely, but not never, taught to outsiders. Most centaurs speak Elvish, Sylvan, or both in addition to their own tongue, and most outsiders who must speak with them do so in one of those tongues. Real world mapping: mostly Greek, but flexible.
+* **Kenku**: Used vaguely Sioux for eastern islands Kenku, but does not need to reflect all kenku.
+
+### Exotic Languages
+Many other creatures in the world of Taelgar speak a language of their own, including many of the unusual or rarer races in the world. These might include the strange language of clicks and hisses the **Giant Eagles** speak, or the strange demonic-influenced barking tongue of **Gnolls.** Most of these are difficult, if not impossible, to learn to speak without magic (and few have a written alphabet).
+
+* **Draconic:** The language of dragons. Few learn the pure, **High Draconic** tongue of the ancient wyrms of old, although it is said to be a powerful language of magic. More commonly heard in Taelgar is **Low Draconic**, the native tongue of kobolds and lesser dragons. The nuances and magical power of High Draconic is usually lost on those who only know Low Draconic, although they are usually mutually comprehensible for basic communication. 
+* **Deep Speech:** This language is more a rumor than a known fact, but is suspected to be the tongue of the alien aberrations of the underdark, notably the mind flayers, their slaves and the twisted creations. Difficult to learn.
+* **Gnoll**: a hideous combination of Abyssal and a magical beast tongue of hyenas; can attempt to understand if you speak Abyssal
+* **Bullywug**: spoken by bullywugs mostly
+* **Grell**: a corrupted form of Deep Speech
+
+### Extra-Planar Languages
+Time and space do not work on the other planes as they do on the material plane, and each plane tends to be represented by a single language. These are typically difficult, although not impossible, for outsiders to learn.
```

Comment:
I think these should actually not be too difficult to learn. Many Tyrwinghans might know a little Sylvan, for example, and it would certainly not be unheard of for priests to know or learn Celestial. 

Some of these are probably not really languages in a traditional sense. For sure Celestial doesn't really make sense as a language in Taelgar - I think it is better thought of as a kind of resonance/echo/understanding of the Divine Essence or something. Need to think about this a bit more.